src/server/user-request-handler.js

```diff
@@ -59,6 +59,32 @@ class UserRequestHandler {
 
     res.send({ profile: savedProfile });
   }
+
+  async handlePostUserPasswordResetRequest(req, res) {
+    const { email } = req.body;
+    const user = await this.userService.getUserByEmailAddress(email);
```

Comment:
I would get here by the user._id on the request (I think I saw this used somewhere else in the code).

Reply:
> I would get here by the user._id on the request (I think I saw this used somewhere else in the code).

There is no _id at this point, since it's the password reset request which is done outside of the user session. That's why we need to obtain the user first, by the provided email.